base/jaeger/jaeger.Deployment.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     app.kubernetes.io/component: jaeger
     app: jaeger
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/component: all-in-one
```

Comment:
@uwedeportivo Are you sure that jaeger isn't relying on this?

Reply:
i'm not sure. i googled a little bit and didn't find anything. the jaeger k8s operator might use it. not sure

Reply:
Okay. We should test to make sure that this change is safe .

Reply:
i tested with a test cluster and it seems fine. jaeger pods came up, i could hit the UI